core/build.gradle

```diff
@@ -149,7 +149,6 @@ task integrationTestCosmos(type: Test) {
     options {
         systemProperties(System.getProperties().findAll{it.key.toString().startsWith("scalardb")})
     }
-    maxParallelForks = 3
```

Comment:
Concurrent operations seem to increase the possibility of tentative errors.

Reply:
Ah, okay. How long does this integration test take? Is it shorter than before?

Reply:
@brfrn169 About 26 minutes.
<img width="292" alt="image" src="https://github.com/scalar-labs/scalardb/assets/59043/775b1a2e-0c03-4605-b3bd-67a790a017b2">

The test once passed with `maxParallelForks = 3` and took 19 minutes. But the test often failed with the configuration...

Reply:
Okay... Thank you. I wish it had solved the slow integration test problem as well. 😅

Reply:
Using larger runners might make parallel execution feasible. It should be way more inexpensive than Cosmos DB, so it is worth trying. Let's discuss it later.